src/init.cpp

```diff
@@ -1631,8 +1631,8 @@ bool AppInitMain(InitInterfaces& interfaces)
 
     // ********************************************************* Step 11: import blocks
 
-    if (!CheckDiskSpace() && !CheckDiskSpace(0, true))
-        return false;
+    if (!CheckDiskSpace(GetDataDir()) && !CheckDiskSpace(GetBlocksDir()))
```

Comment:
Should this not be `||`? /cc @jonasschnelli 386a6b62a8a1db9dd0f354cb95b7585f555c7e5d

Reply:
Yeah, seems like it to me.

Reply:
#15124